CONTRIBUTING.md

```diff
@@ -144,6 +144,21 @@ Here is a recommended way to get setup:
 6. Write code, open a PR from your branch when you're ready
 7. If you need to rebase your fork's PR branch onto master to resolve conflicts: `git fetch upstream`, `git rebase upstream/master` and force push to Github `git push --force origin your-branch`
 
+## Changelog Conventions
+
+What warrants a changelog entry?
```

Comment:
@mapbox/gl-js The list is not exhaustive, feel free to suggest any other suggestion to add/remove here